api/v1alpha1/meta_types.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2024 Nutanix. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+// TypedLocalObjectReference contains enough information to let you locate the
+// typed referenced object inside the same namespace.
+// This is redacted from the upstream https://pkg.go.dev/k8s.io/api/core/v1#TypedLocalObjectReference
```

Comment:
nit: I think this is a typo.

```suggestion
// This is copied from the upstream https://pkg.go.dev/k8s.io/api/core/v1#TypedLocalObjectReference
```

Reply:
it was intentional, we are using only 2 fields from upstream struct so used `redacted`